src/templates/pt/2019/base.html

```diff
@@ -0,0 +1,203 @@
+  "ScottHelme": 'Scott Helme é um pesquisador de segurança e fundador da <a href="https://report-uri.com">report-uri.com</a> e <a href="https://securityheaders.com">securityheaders.com</a>. Você pode encontrá-lo falar sobre segurança no Twitter <a href="https://twitter.com/Scott_Helme">@Scott_Helme</a> e bloga em <a href="https://scotthelme.co.uk">scotthelme.co.uk</a>.',
+  "tammyeverts": 'Tammy Everts passou mais de duas décadas estudando usabilidade e UX. Nos últimos dez anos, ela se concentrou na interseção do UX com desempenho e negócios da Web. Ela é CXO em <a href="https://speedcurve.com/">SpeedCurve</a>, co-presidente do <a href="https://perfnow.nl/">performance.now() conference</a>, e autor do livro O\ 'Reilly <em><a href="http://shop.oreilly.com/product/0636920041450.do">Time Is Money: The Business Value of Performance</a></em>.',
+  "tomayac": 'Thomas Steiner é um advogado de desenvolvedores da Web no Google Hamburg, focado em tornar a Web um lugar melhor por meio da padronização, criação e compartilhamento de melhores práticas e pesquisa. Ele bloga em <a href="https://blog.tomayac.com/">blog.tomayac.com</a> e os tweets como <a href="https://twitter.com/tomayac">@tomayac</a>.',
+  "una": 'Una Kravets é palestrante pública internacional do Brooklyn, redatora técnica e advogada de desenvolvedores de design de materiais no Google. A Una hospeda <a href="https://www.youtube.com/watch?v=YK8GZBx3hpg">Designing the Browser</a> e a série <a href="https://spec.fm/podcasts/toolsday">Toolsday</a> podcast de desenvolvedor . Siga-a no <a href="https://twitter.com/una">Twitter</a> para encontrar suas reflexões sobre CSS criativo, experiências do usuário e práticas recomendadas de desenvolvimento da web.',
+  "ymschaap": 'Fundador técnico da consultoria SEO  <a href="https://build.amsterdam/">build.amsterdam</a>. Anteriormente, fundou várias empresas da web que atingiram mais de 1 bilhão de usuários. Blogar sobre suas últimas (ad) ventures desde 2005 em <a href="https://yvoschaap.com/">yvoschaap.com</a>.',
+  "zachleat": 'Zach é um desenvolvedor Web com <a href="https://www.filamentgroup.com/">Filament Group</a>. Ele está atualmente fixado em <a href="https://www.zachleat.com/web/fonts/">web fonts</a> e <a href="https://www.zachleat.com/web/introducing-eleventy/">static site generators</a>. Sua <a href="https://www.zachleat.com/web/speaking/">public speaking résumé</a> inclui palestras em oito países diferentes em eventos como JAMstack_conf, Beyond Tellerrand, Smashing Conference, CSSConf, e <a href="https://www.zachleat.com/web/whitehouse/">The White House</a>. Ele também ajuda em <a href="http://nejsconf.com/">NEJS CONF</a> e o <a href="http://nebraskajs.com">NebraskaJS</a> meetup.',
```

Comment:
Should we translate "static site generators"?

And meetup? Or is that a well-knownused phrase in Portuguese now?

Reply:
I believe that some terms should not be translated, but if you want I can translate without problems.